calendar-amqp/src/main/java/com/linagora/calendar/amqp/EventAlarmHandler.java

```diff
@@ -74,12 +75,27 @@ public EventAlarmHandler(AlarmInstantFactory alarmInstantFactory,
         this.eventEmailFilter = eventEmailFilter;
     }
 
+    public Mono<Void> handleCreate(CalendarAlarmMessageDTO alarmMessageDTO) {
+        if (!hasVALARMComponent(alarmMessageDTO)) {
+            return Mono.empty();
+        }
+        return handleCreateOrUpdate(alarmMessageDTO);
+    }
+
     public Mono<Void> handleCreateOrUpdate(CalendarAlarmMessageDTO alarmMessageDTO) {
         return openPaaSUserDAO.retrieve(alarmMessageDTO.extractCalendarURL().base())
             .filterWhen(openPaaSUser -> isUserAlarmEnabled(openPaaSUser.username()))
             .flatMap(openPaaSUser -> processCreateOrUpdate(openPaaSUser.username(), alarmMessageDTO));
     }
 
+    private boolean hasVALARMComponent(CalendarAlarmMessageDTO alarmMessageDTO) {
+        JsonNode event = alarmMessageDTO.calendarEvent();
+        return event != null
+            && event.isArray()
+            && event.size() > 2
+            && event.get(2).toString().contains("\"valarm\"");
```

Comment:
I would need more explanation on this heurustic: waht does 2 means? 
What is the expected list of components?